eng/icu.proj

```diff
@@ -37,4 +37,22 @@
             Lines="@(_ItemsFromEmscriptenVersionFile->Replace('&quot;', ''))"
             Overwrite="true" />
   </Target>
+  <Target Name="PatchWasiSdk" Condition="'$(TargetOS)' == 'Wasi'">
+    <ReadLinesFromFile File="../.devcontainer/wasi-sdk-version.txt">
+    <Output TaskParameter="Lines" ItemName="_VersionLines" />
+    </ReadLinesFromFile>
+
+    <PropertyGroup>
+      <WASI_SDK_PATH>$(RepoRoot)\artifacts</WASI_SDK_PATH>
+      <WasiLocalPath>$(RepoRoot)\artifacts</WasiLocalPath>
+      <WasiSdkVersion>%(_VersionLines.Identity)</WasiSdkVersion>
+      <WasiSdkUrl>https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-$(WasiSdkVersion)/wasi-sdk-$(WasiSdkVersion).0-linux.tar.gz</WasiSdkUrl>
```

Comment:
These properties are only for private use, so prepend `_` to the names like `_WasiLocalPath`.